openquake/calculators/scenario_risk.py

```diff
@@ -47,20 +47,26 @@ def scenario_risk(riskinputs, riskmodel, rlzs_assoc, monitor):
     :param monitor:
         :class:`openquake.baselib.performance.PerformanceMonitor` instance
     :returns:
-        a dictionary (key_type, loss_type) -> losses where the `key_type` can
-        be "agg" (for the aggregate losses) or "ins" (for the insured losses).
+        a dictionary {
+        'agg': array of shape (E, R, 2),
```

Comment:
You are too careful! I updated the docstring with an obsolete version of the code :-(
Now it is fine.